src/locale/en.ts

```diff
@@ -56,4 +56,16 @@ export const en = {
     none: 'None',
     latest: 'Latest transaction',
   },
+  settings: {
```

Comment:
locale in `src/locale/en` is essentially hardcoded - there's no mechanism to change it.
Use `i18n` and put your string values to `loc/en.js` :)

Reply:
Yup, I know but I thought that we would implement it in the future :thinking: Okay so I'm gonna fix  :+1: